pkg/consoleuser/consoleuser_windows.go

```diff
@@ -0,0 +1,87 @@
+//go:build windows
+// +build windows
+
+package consoleuser
+
+import (
+	"context"
+	"fmt"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/shirou/gopsutil/process"
+)
+
+func CurrentUids(context context.Context) ([]string, error) {
+	explorerProcs, err := explorerProcesses()
+	if err != nil {
+		return nil, fmt.Errorf("getting explorer processes: %w", err)
+	}
+
+	// unclear if windows will ever have more than one explorer process for a single user
+	// guard against this by forcing uniquness
+
+	// first store uids in a map to prevent duplicates
```

Comment:
I am always baffled that golang doesn't have succinct sugar for "get me all the unique items". But I agree this seems fine.